elections/steering/2022/results.md

```diff
@@ -0,0 +1,18 @@
+# Results of the 2022 Steering Committee Election
```

Comment:
It looks like we skipped it for last year's election (think it was an accident) - but in the past we've published the full list in the event we do need to fill an additional seat:
- https://github.com/kubernetes/community/blob/master/elections/steering/2020/RESULTS.md#results
- https://github.com/kubernetes/community/blob/master/elections/steering/2019/RESULTS.md#results
- https://github.com/kubernetes/community/blob/master/elections/steering/2018/RESULTS.md#results

Reply:
Maybe do the next one or two? I don't want the results to go over into hurt feelings territory if we can help it.

Reply:
@coderanger dunno, i feel like picking full list vs only winners (not winners + a couple more)

Reply:
Full list for transparency, please.